packages/react-components/src/components/VideoGallery/RemoteScreenShare.tsx

```diff
@@ -92,11 +92,27 @@ export const RemoteScreenShare = React.memo(
       return (
         <VideoTile
           renderElement={
+            // Add the layer on top of the pptlive stream to diable keyboard event
```

Comment:
Can this happen at the SDK level? Otherwise every developer using pptlive will have to do this

Reply:
Or does the SDK support navigating the PPTLive?

Reply:
I believe this is more like a "hotfix" instead of a long term fix

@ShaunaSong - could you please create a long term fix work item and put it here? I believe that is the back to presenter button one, let's keep it in track before we lose it

Reply:
I'm not certain this should ever be the default behavior.
We can't ship a "sync to presenter" button through the SDK as it would have to be localized.
And we can't anticipate every developer having to implement that either.
So any developer who just wants to display the pptLive stream will have to somehow know to create this hacky fix, and likely won't know, instead they'll get bugs that say that a user was no longer in sync with the presenter and have no idea how a user could be in that state.

I would propose moving this logic to the SDK, then when the SDK supports navigation controls, we expose pptLive.feature.nextSlide()/previousSlide()/syncToPresenter() functions a developer can call _if_ they want this behavior

Reply:
As we discuss in feature sync up we decide to have the overlay here as the temple fix. Will discuss more with Calling SDK team, how we could do about this fix.

Reply:
Sorry missed the sync, would the fix not be exactly the same but at the calling SDK layer on the target element?

Reply:
Discussed offline, we will check this in to unblock UI library stable release, then apply the same fix to the SDK layer and can remove this once that fix progresses to UI library